### PR TITLE
Update asynchronous processing in agents

### PR DESCRIPTION
**Pull request by nqyy**
- Update async calling.
- Doc and version updates
- Remove some unnecessary files.

### CODE REVIEW

File: mbodied/agents/backends/gradio_backend.py [hunk 1]
```diff
@@ -51,9 +53,10 @@ def async_act(self, *args, api_name="/predict", result_callbacks=None, blocking=
             Job: Gradio job object.
         """
         job = self.client.submit(api_name=api_name, result_callbacks=result_callbacks, *args, **kwargs)
-        tic = time.time()
         if blocking:
+            tic = time.time()
             while not job.done:
```
Comment (sebbyjp) [inline mbodied/agents/backends/gradio_backend.py:58]: just call job.result(timeout=timeout) if its blocking.

File: mbodied/agents/motion/motor_agent.py [hunk 2]
```diff
@@ -35,3 +35,14 @@ def act(self, **kwargs) -> Motion:
...
+    async def async_act(self, **kwargs) -> Motion:
+        """Generate a Motion asynchronously based on given parameters.
+
+        Args:
+            **kwargs: Arbitrary keyword arguments for motor agent to act on.
+
+        Returns:
+            Motion: A Motion object based on the provided arguments.
+        """
+        pass
```
Comment (sebbyjp) [inline mbodied/agents/motion/motor_agent.py:48]: raise NotImplemented

File: mbodied/agents/motion/motor_agent.py [hunk 3]
```diff
@@ -35,3 +35,14 @@ def act(self, **kwargs) -> Motion:
             Motion: A Motion object based on the provided arguments.
         """
         pass
```
Comment (sebbyjp) [inline mbodied/agents/motion/motor_agent.py:37]: raise NotImplemented here too

File: README.md [hunk 4]
```diff
@@ -32,7 +32,7 @@
 
 **Updates:**
 
-**June 30 2024, embodied-agents v1.0.0**:
+**June 30 2024, embodied-agents v1.0**:
```
Comment (sebbyjp) [inline README.md:35]: increment to match pypi?

**Review by sebbyjp — COMMENTED** [review]
Why do we have to reimplement async_act so frequently? Can we just have a base implementation that
1. uses gradio client logic if backend is gradio client
2.  defaults to calling act() in asyncio thread else

that way subclasses can choose to override it for custom behavior and get it for free with just implementing act()

File: README.md (hunk 4 above)
Comment (nqyy) [reply README.md:35]: Nah, we may want to have minor updates/fixes like v1.0... This should just be the general version.

File: mbodied/agents/backends/gradio_backend.py (hunk 1 above)
Comment (nqyy) [reply mbodied/agents/backends/gradio_backend.py:58]: This method should not have a blocking at all. Updated and also renamed.

**Review by sebbyjp — APPROVED** [review]

**Review by sebbyjp — APPROVED** [review]

**Pull request merged**